### ISSUE TITLE
Add a .editorconfig

### ISSUE THREAD

**Issue by NickStallman** [pelias/pelias#791]
Hey team! 

Would it be possible to add a .editorconfig to the project's repos? https://editorconfig.org/

Indenting using spaces with a width of 2 isn't my normal syntax so my editor doesn't like it.
I've made my own little .editorconfig to make it easier for me, but adding it to the repositories would make it easier for others to develop and it's used by basically all IDEs.

Thanks

**Comment by missinglink** [comment]
Works for me, i actually been thinking of enforcing a code style.

I've been against the idea for a long time but I'm just getting sick of formatting/reformatting code, it's one of the things I love about Go, I just use the formatting tools and it's all does for me.

I've been working on a new repo which I'll hopefully publish this week as `pelias/parser`, for that repo I adopted something called `standardjs` which I personally don't love but I'm ok using because it's consistent.

Anyway, sorry I highjacked your issue, I see now reason not to merge PRs adding an editor config as long as it's well defined and consistent.

**Comment by orangejulius** [comment]
I'd support this as well, since my editor alerts me when there is a missing newline at the end of a file and...ahem certain other people...do not have their editor configured the same way :P

It's annoying to have to keep config files like this in sync across all our repos, but as long as the first version is reasonably correct I think we'll be ok.

Two spaces for `.js` and `.yml`, newline at ends of file, what else do we want?

**Comment by NickStallman** [comment]
Yeah I'm weird and my normal style is tabs with 8 spacing so having my editor switch in to the correct style is extremely handy. :)

standardjs removing semicolons? Ouch that'll take some getting used to.
Otherwise that's cool for enforcing consistency.
.editorconfig will still be useful in getting it in to the correct format before standardjs takes over.

I did a quick config file here for the openstreetmap repo: https://github.com/pelias/openstreetmap/pull/477
It seems to behave correctly in my editor.

**Comment by missinglink** [comment]
Haha right, there is also https://github.com/Flet/semistandard ;)
